pyproject.toml

```diff
@@ -99,7 +99,7 @@ markers = "platform_python_implementation == 'CPython'"
 
 [tool.pytest.ini_options]
 markers = "slow: Mark the test as slow. Can be skipped with --skip_slow"
-addopts = "--no-cov-on-fail --cov=manim --cov-report xml --cov-report term -n auto"
+addopts = "--no-cov-on-fail --cov=manim --cov-report xml --cov-report term -n auto --dist=loadfile"
```

Comment:
@naveen521kk I added this argument as it seems like it would allow parallelization but, for tests in the same file they would share a worker to avoid issues with multiple opengl windows running at the same time. Does this make sense to you? got the information here - https://stackoverflow.com/questions/4637036/is-there-a-way-to-control-how-pytest-xdist-runs-tests-in-parallel

Reply:
Looks good